packages/bpk-component-split-input/README.md

```diff
@@ -0,0 +1,43 @@
+# bpk-component-split-input
+
+> Backpack split-input component.
+
+## Installation
+
+```sh
+npm install bpk-component-split-input --save-dev
+```
+
+## Usage
+
+```js
+import React from 'react';
+import BpkSplitInput from 'bpk-component-code';
+import { INPUT_TYPES } from 'bpk-component-input';
```

Comment:
I am a bit unclear on this. The `bpk-component-input` needs to be installed since it's also used in the `BpkInputField` and also for the INPUT_TYPES here. Do you mean to declare and export INPUT_TYPES independetly from `BpkSplitInput` index similarly to what we do in BpkInput?